lib/rmt/config.rb

```diff
@@ -3,15 +3,34 @@
 
 Config.setup do |config|
   config.merge_nil_values = false
+
+  unless ENV['RAILS_ENV'] == 'test'
+    config.schema do
+      required(:scc).schema do
+        required(:username).filled
+        required(:password).filled
+      end
+    end
+  end
 end
 
 Config.load_and_set_settings(
+  File.join(__dir__, '../../config/default_rmt.yml'),
```

Comment:
My concern would be that this file is can be modified by the users.

Reply:
Could you suggest a better place for this file, or a way that the user couldn't modify? :)

Reply:
Yeah, I'm not sure -- I'm still thinking about this. Apparently, `dry-validation` doesn't allow specifying default values. However, `dry-initializer` does -- perhaps it would be nicer to have explicitly defined config methods, rather than relying on `method_missing?`. But I'm not sure how well it handles nested config keys (as they are currently defined) and how well that would work in general. :-)